docs/src/operator/monitoring.md

```diff
@@ -74,7 +74,26 @@ block_builder.build_block
 
 ### Batch building
 
-Not yet implemented.
+This trace covers the building and proving of a batch.
+
+<details>
+  <summary>Span tree</summary>
+
+```sh
+batch_builder.build_batch
+┝━ batch_builder.wait_for_available_worker
+┝━ batch_builder.select_batch
+│  ┝━ mempool.lock
+│  ┕━ mempool.select_batch
+┝━ batch_builder.get_batch_inputs
+│  ┕━ store.client.get_batch_inputs
+┝━ batch_builder.propose_batch
+┝━ batch_builder.prove_batch
+┝━ batch_builder.inject_failure
+┕━ batch_builder.commit_batch
+   ┝━ mempool.lock
+   ┕━ mempool.batch_proved
```

Comment:
Should this be `mempool.commit_batch` (and should it mention the option of `rollback_batch`)?

Reply:
Ah it should, I updated it afterwards - thank you.

Re: `rollback_batch` - I'm not sure how to really document the failure paths since it can happen at any stage.